### ISSUE TITLE
[QA] Transaction is shown as completed when the status on Wormholescan is still "Pending redeem"

### ISSUE THREAD

**Issue by tsadovska** [wormhole-foundation/wormhole-connect#2908]
## Describe the current behavior in detail
Checked on https://portalbridge.com/.

Steps:
1. Select Fantom network and FTM token in "From" section (MetaMask wallet is auto-connected)
2. Select BSC network and WFTM token in "To" section (MetaMask wallet is auto-connected)
3. Enter Amount, 1
4. Select Automatic route option
5. Click on "Review transaction" button
6. Change slider position to 42%
7. Open link to Wormholescan when transaction is shown as completed on Portal Bridge
8. Check transaction status 

Actual result: Status "pending redeem" is shown.
Expected result: Status is "completed". 

## Notes/ Attachments

After 1-2 minutes, correct status is shown on [Wormholescan](https://wormholescan.io/#/tx/0x45324bc380239c1e1627505c3aefe5ab7c8485eae50ba930e4b45e94bccd9427). Asset is received: [BscScan](https://bscscan.com/tx/0xc4c3bad78ed096ee3bdb85ad660a9d32350a33c9ab93e879fd1d3c33e614787f).

https://github.com/user-attachments/assets/04de325f-e25b-42a2-9d64-a4f39c5809d1

**Comment by Wesleyleung** [comment]
You recieed the asset in your wallet on time right? It looks like only Wormholescan is delayed?

**Comment by tsadovska** [comment]
@Wesleyleung Yes, when token has already been received on destination chain, status "pending redeem" on Wormholescan for a some time. 


https://github.com/user-attachments/assets/204c521b-ea2a-4f55-ad98-6551b619392f

**Comment by agodlevska** [comment]
Issue is no longer reproducible on https://portalbridge.com/, [Wormholescan link](https://wormholescan.io/#/tx/0x72986ff7040d642d5bbcf2dd1fb43437bc76927c300b957589e02d8a70107a04).

Status is "Transaction completed".

Closing the ticket. 

Attachment:

<img width="1440" alt="Image" src="https://github.com/user-attachments/assets/24281170-0a84-4f2b-9019-0e485f4e3a66" />
